app/response_processor.py

```diff
@@ -70,37 +70,37 @@ def process(self, msg, tx_id=None):
 
         self.logger = self.logger.bind(tx_id=self.tx_id)
 
-        response_type = str(decrypted_json.get("type"))
-
         try:
             self.validate_survey(decrypted_json)
         except ClientError:
             # If the validation fails, the message is to be marked "invalid"
             # and then stored. We don't then want to stop processing at this point.
-
             decrypted_json['invalid'] = True
             self.logger.info("Invalid survey data, skipping receipting")
```

Comment:
This log line is a little misleading as it will also skip downstream processing?